### PR TITLE
fix wording for docker/podman drivers to "container"

### LINKED ISSUE #6859: change word "VM" to "container" for kic drivers
when starting a minikube again:
 

🏃  Using the running docker "minikube" VM ...

also when starting a stopped minikube

🔄  Starting existing docker VM for "minikube" ...

### PR DESCRIPTION
**Pull request by nkprince007**
fixes #6859

before:
```
$ minikube start
Using the running docker "minikube" VM ...
```
```
$ minikube start
Starting existing docker VM for "minikube"...
```

after:
```bash
$ minikube start
Using the running docker "minikube" container ...
```
```
$ minikube start
Starting existing docker container for "minikube"...
```

### CODE REVIEW

**Comment by medyagh** [comment]
@nkprince007 can you please put before and after in the PR description?

**Comment by nkprince007** [comment]
Done @medyagh

File: pkg/minikube/driver/driver.go [hunk 1]
```diff
@@ -85,6 +85,14 @@ func Supported(name string) bool {
 	return false
 }
 
+// MachineType returns appropriate machine name for the driver
+func MachineType(name string) string {
```
Comment (josedonizetti) [inline pkg/minikube/driver/driver.go]: @nkprince007 can we have a test for this func? :)

Comment (nkprince007) [reply pkg/minikube/driver/driver.go]: Sure :)

File: test/integration/driver_type_test.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
+// +build integration
```
Comment (medyagh) [inline test/integration/driver_type_test.go]: There was no need to add integraiton test to test this,

please remove this and add a unit test instead.

Comment (nkprince007) [reply test/integration/driver_type_test.go]: yup, thanks for the suggestion :)

**Pull request merged**